eng/Versions.props

```diff
@@ -95,4 +84,83 @@
     <MicrosoftCodeAnalysisVersion>4.8.0-2.23422.14</MicrosoftCodeAnalysisVersion>
     <MicrosoftNetCompilersToolsetVersion>4.8.0-2.23422.14</MicrosoftNetCompilersToolsetVersion>
   </PropertyGroup>
+  <!-- 
+    SOS test runtime versions
+
+    Internal service release testing:
+
+    $(DotnetRuntimeVersion)         - the service release version to test against (fx-version option value) i.e. 2.1.17, 3.1.3 or "default"
+    $(DotnetRuntimeDownloadVersion) - the service release package version i.e. 2.1.17, 3.1.3-servicing.20128.1 or "default"
+    $(RuntimeSourceFeed)            - the service release internal blob storage link
+    $(RuntimeSourceFeedKey)         - the service release blob feed token
+     
+  -->
+  <PropertyGroup>
+    <MicrosoftNETCoreApp80Version>8.0.2</MicrosoftNETCoreApp80Version>
+    <MicrosoftNETCoreApp70Version>7.0.16</MicrosoftNETCoreApp70Version>
+    <MicrosoftNETCoreApp60Version>6.0.27</MicrosoftNETCoreApp60Version>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <DotnetRuntimeVersion Condition="'$(DotnetRuntimeVersion)' == ''">default</DotnetRuntimeVersion>
+    <InternalReleaseTesting>false</InternalReleaseTesting>
+    <InternalReleaseTesting Condition="'$(DotnetRuntimeVersion)' != 'default'">true</InternalReleaseTesting>
+    <ExtraInstallArgs>-runtimesourcefeed '$(RuntimeSourceFeed)' -runtimesourcefeedkey '$(RuntimeSourceFeedKey)'</ExtraInstallArgs>
+  </PropertyGroup>
+
+  <ItemGroup Condition="!$(InternalReleaseTesting)">
+    <RuntimeTestVersions Include="Latest">
+<!--
+      <RuntimeDownload>$(VSRedistCommonNetCoreSharedFrameworkx6490Version)</RuntimeDownload>
+      <Runtime>$(MicrosoftNETCoreAppRuntimewinx64Version)</Runtime>
+      <AspNetDownload>$(MicrosoftAspNetCoreAppRefInternalVersion)</AspNetDownload>
+      <AspNet>$(MicrosoftAspNetCoreAppRefVersion)</AspNet>
+-->
```

Comment:
Is this needed?

Reply:
Yes, it will be uncommented when we (Juan) switches the DARC update channel to 9.0.